var/spack/repos/builtin/packages/py-cupy/package.py

```diff
@@ -35,20 +36,24 @@ class PyCupy(PythonPackage, CudaPackage, ROCmPackage):
     depends_on("py-fastrlock@0.5:", type=("build", "run"))
     depends_on("py-numpy@1.20:1.25", when="@:11", type=("build", "run"))
     depends_on("py-numpy@1.20:1.26", when="@12:", type=("build", "run"))
+    depends_on("py-numpy@1.22:1.29", when="@13:", type=("build", "run"))
```

Comment:
Version ranges are inclusive (https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges) so this should be:

```suggestion
    depends_on("py-numpy@1.22:1.28", when="@13:", type=("build", "run"))
```

Reply:
should be fixed now